### ISSUE TITLE
[Feature] Portable (no-install) version. 

### ISSUE THREAD

**Issue by sapioit** [lencx/ChatGPT#366]
### Feature description

Please make a Portable (no-install) version which stores settings and configs inside the installation folder, instead of `%appdata%`, the registry, and other locations.

### Motivation

Portability. Being able to simply place it on an USB stick and have it with you everywhere you have PC access, even if you cannot install apps on said PC. Just download the archive, un-archive it on an USB stick or anywhere else on your PC, then use it. 

### Alternatives

If you don't want to make a portable version yourselves, please talk with the guys at https://portableapps.com to make a portable version of your app for you and distribute it themselves, or at least to give them permission to make a portable version of your app and distribute it themselves.

### Additional context

If you give PortableApps.com permission to make a portable version of your app, you might not get statistics about the app downloads unless you ask them for it, you at least get more popularity by having your app listed in their app directory, which many people use to search for apps they don't know about but would be interested in trying (since they're portable and leave no traces of usage once closed and deleted).

**Comment by lencx** [comment]
Currently all data for the application is stored in the `~/.chatgpt` directory. You can zip this directory directly and take it with you.

**Comment by MandiYang** [comment]
There is something called portable mode that you could enable when you download a Appimage.
https://docs.appimage.org/user-guide/portable-mode.html

**Comment by Manamama** [comment]
Or one could use one's account via any browser (also on mobile): as portable as it gets! 

FYI, I use @lencx solution as a handy "dedicated browser three" as it also refreshes the connection (see his scripting advice somewhere about the "Inject Scripts" bit) so my "browser two and browser one" have some other accounts, also given the throttling and commercializing (aka "monetizing") of the OpenAI engine last week.

**Comment by olljanat** [comment]
Not exactly portable but FYI that I did publish MSI package which can be installed without local admin rights on https://github.com/olljanat/ChatGPT 

Config and exported markdown files I pushed to GitHub private repo so it is easy to sync those between computers (PR https://github.com/lencx/ChatGPT/pull/827 is also related to that scenario).

**Comment by navyfighter12** [comment]
> Not exactly portable but FYI that I did publish MSI package which can be installed without local admin rights on https://github.com/olljanat/ChatGPT
> 
> Config and exported markdown files I pushed to GitHub private repo so it is easy to sync those between computers (PR #827 is also related to that scenario).

MVP! This is what I am talking about. Installed and runs perfectly without needing to get admin rights. Thank you so much for this

**Comment by Enlight432** [comment]
Yeah, i need the portable version because I want to open two accounts simultaneously

**Comment by dg2k** [comment]
A portable version is really a great addition.
